tests/test_client.py

```diff
@@ -307,6 +310,20 @@ def test_no_search_link(self, api):
             api.search(limit=10, max_items=10, collections='naip')
         assert 'No link with "rel" type of "search"' in str(excinfo.value)
 
+    def test_no_conforms_to(self):
+        with open(str(TEST_DATA / 'planetary-computer-root.json')) as f:
+            data = json.load(f)
+        del data["conformsTo"]
+        with TemporaryDirectory() as temporary_directory:
+            path = os.path.join(temporary_directory, "catalog.json")
+            with open(path, "w") as f:
+                json.dump(data, f)
+            api = Client.from_file(path)
```

Comment:
This tom-foolery is necessary because the conformance is set up during `Client` instantiation, and I didn't want to add a new test file just to remove one field.